coding-concepts.js

```diff
@@ -8,44 +8,57 @@
 // --------------------1) What will this log?
 
 var cohort = "Delta 2021"
-// console.log(cohort.length)
+console.log(cohort.length)
 
 // a) Your answer:
-// b) Verify and explain:
+// 10
 
+// b) Verify and explain:
+// Initial answer was correct // Counted every character and space in the var cohort variable string to end up with a length of 10 to print on the console // Started counting length at 1
 
 // --------------------2) What will this log?
 
 var greeting = "Hello World!"
-// console.log(greeting[3])
+console.log(greeting[3])
 
 // a) Your answer:
+// l
+
 // b) Verify and explain:
+// Initial answer was correct // greeting[3] in console.log will print the character at index 3 of the string // started index count at 0
 
 
 // --------------------3) What will this log?
 
 var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
 // a) Your answer:
-// b) Verify and explain:
+// Ruby
 
+// b) Verify and explain:
+// Initial answer was correct // console.log will print the value at index 1 in the var languages array // started index count at 0
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
 // a) Your answer:
-// b) Verify and explain:
+// SATURDAY
+// SUNDAY
 
+// b) Verify and explain:
+// Initial answer was wrong // terminal returned an error message // toUpperCase function ran into a problem attempting to uppercase the array // javascript requires more info before it can perform this function in an array
```

Comment:
Cool! so the interesting catch here is that .toUpperCase() is a string method and won't work on an array.